File: SampleSetup/Output/Swift/Body/ApiTokenPostArg.swift
struct ApiTokenPostArg: Codable {
    @available(*, deprecated, message: "Use the other one")
    var UserName: String = ""

    /**
     * A sample summary.
     */
    var UserNameKind: Int = 0

    /**
     * A multiline summary.
     * test explanations :)
     */
    var Password: String = ""
}

File: SampleSetup/Output/PHP/Body/ApiTokenPostArg.php
<?php
namespace SampleProject\Protocol\Body;

class ApiTokenPostArg {
    /**
     * @deprecated Message: Use the other one
     * @var string
     */
    public $UserName;

    /**
     * A sample summary.
     * @var int
     */
    public $UserNameKind;

    /**
     * A multiline summary.
     * test explanations :)
     * @var string
     */
    public $Password;

    public function __construct()
    {
        $this->UserName = "";
        $this->UserNameKind = 0;
        $this->Password = "";
    }
}
